src/renderer/windows/GPMWebView/playback/controller.js

```diff
@@ -45,20 +45,28 @@ Emitter.on('playback:stop', () => {
 
 Emitter.on('playback:thumbsUp', () => {
   if (!remote.getGlobal('PlaybackAPI').data.song.title) return;
-  new Notification('You just liked', { // eslint-disable-line
-    body: remote.getGlobal('PlaybackAPI').data.song.title,
-    icon: remote.getGlobal('PlaybackAPI').data.song.albumArt,
-  });
-  window.GPM.rating.setRating(5);
+  if (window.GPM.rating.getRating() !== '5') {
```

Comment:
Not 100% sure how this change is relevent or what it does?  Can you either clarify or remove ❓

Reply:
Before, the toast got fired every time the menu option was clicked - which was fine because once a song was already liked clicking the button again didn't do anything.
The menu option is now a toggle, so depending on whether the song is currently liked or not the rating either needs to be reset ("un-liked", as distinct from disliked) or set to 5.
Only in the latter case should the "you just liked..." toast be shown. I didn't add an "un-liking" toast because I wasn't sure if that was the correct term to use.

(same argument in reverse applies to the dislike option)